src/listProjects.ts

```diff
@@ -0,0 +1,56 @@
+import * as api from "./api";
+import * as FirebaseError from "./error";
+import * as logger from "./logger";
+
+const ONE_SECOND_MILLIS = 1000;
```

Comment:
Similar to the previous PR, this could just be TIMEOUT_MILLIS.

Reply:
Done.